Package.swift

```diff
@@ -4,19 +4,56 @@ import PackageDescription
 let package = Package(
     name: "UtiliKit",
     platforms: [
-        .iOS("10.0")
+        .iOS("11.0")
```

Comment:
Is there a technical reason for updating to iOS 11 or did you just feel that it was time?

Reply:
Yea I originally had done so because `SnapshotTesting` upped theirs (https://github.com/pointfreeco/swift-snapshot-testing/blob/main/Package.swift) and I was getting an issue adding that package properly. 

Now that I know there is currently way to to denote 'test only dependency' in SPM, and so we need to continue to use Carthage for dependencies (at least until we no longer have the need for `SnapshotTesting`), I'll revert this change